.travis.yml

```diff
@@ -31,14 +37,7 @@ install:
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger danger-junit; fi'
 
 script:
-  - docker-compose exec web bash -c "wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
-  - docker-compose exec web bash -c "dpkg -i google-chrome-stable_current_amd64.deb; apt-get -fy install"
-  - docker-compose exec web bash -c "wget https://chromedriver.storage.googleapis.com/74.0.3729.6/chromedriver_linux64.zip" # https://sites.google.com/a/chromium.org/chromedriver/
-  - docker-compose exec web bash -c "unzip chromedriver_linux64.zip"
-  - docker-compose exec web bash -c "mv chromedriver /usr/local/bin/chromedriver"
-  - docker-compose exec web bash -c "chmod +x /usr/local/bin/chromedriver"
-  - docker-compose exec web bash -c "CI=TRUE GENERATE_REPORT=true rake test:all"
-  - docker-compose exec web bash -c "CI=TRUE GENERATE_REPORT=true rails test:system test"
+  - docker-compose exec web bash -c "CI=TRUE GENERATE_REPORT=true $TASK"
   - echo -e '<?xml version="1.0" encoding="UTF-8"?>' > output.xml
   - tail -n +2 -q ./test/reports/TEST*.xml >> output.xml
```

Comment:
Ah! I see the reports folder is being deleted on this line - so there may be an ordering thing going on here:

https://travis-ci.org/publiclab/plots2/jobs/535281058#L5144

Reply:
@jywarren I think I might know what's going on, this command only makes sense when we just ran the system tests right? So since I have slip those commands execution it will brake when it runs in normal tests.  I'll fix that.

Reply:
yes, i think that's right. Although junit was originally for regular, not system tests. I honestly don't remember where we put the folder deletion line, but we can probably use search to find it!